playbooks/openshift/delete-cluster.yml

```diff
@@ -4,6 +4,9 @@
 #  roles:
 #  - ../../galaxy/openshift-ansible-contrib/roles/openshift-pv-cleanup
 
+- pause:
+    prompt: "This playbook will delete a cluster. Continue if you are sure you want to this or abort the playbook if this is not your intent"
```

Comment:
Per PR #274, please update this prompt to read `"Are you sure you want to delete the {{env_id}} cluster (yes/no)"`